### PR TITLE
enh(UI): Reduce space inside expansion panel summary and details

### PR DESCRIPTION
**Pull request by Thebarda**

### CODE REVIEW

**Comment by bdauria** [comment]
@Thebarda there is something weird in the right bar, it seems the border continues until the bottom, while the panel actually stops before (see the storyshot)

File: src/RightPanel/index.jsx [hunk 1]
```diff
@@ -148,15 +148,17 @@ const RightPanel = ({
...
-          <SecondaryPanelBar
-            aria-label="Close Secondary Panel"
-            display="flex"
-            alignItems="center"
-            alignContent="center"
-            onClick={toggleSecondaryPanel}
-          >
-            {secondaryPanelActive && <CloseSecondaryPanelIcon />}
-          </SecondaryPanelBar>
+          {secondaryPanelActive && (
```
Comment (bdauria) [inline src/RightPanel/index.jsx:151]: The panel bar should always be visible, regardless of the secondary panel is active or not

**Comment by jdelpierre** [comment]
> here is something weird in the right bar, it seems the border continues until the bottom, while the panel actually stops before (see the storyshot)

I have the same remark, is that normal?

**Comment by Thebarda** [comment]
> > here is something weird in the right bar, it seems the border continues until the bottom, while the panel actually stops before (see the storyshot)
> 
> I have the same remark, is that normal?

I fixed that

File: src/RightPanel/index.jsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ const MainPanel = styled(Box)({
 const SecondaryPanelBar = styled(Box)({
   border: '1px solid #D6D6D8',
   width: 20,
-  height: '100%',
   cursor: 'pointer',
+  marginBottom: '49px',
```
Comment (bdauria) [inline src/RightPanel/index.jsx:61]: Is this supposed to correspond to the height of the HeaderContainer? in that case, we need to extract a variable to synchronize both values

Comment (bdauria) [inline src/RightPanel/index.jsx:61]: Is this supposed to correspond to the height of the HeaderContainer? in that case, we need to extract a variable to synchronize both values

**Pull request merged**